examples/test.py

```diff
@@ -3,20 +3,20 @@
 import sys, os
 sys.path.append("../")
 
-from zfit.core import optimisation as opt
+from zfit.core import optimization as opt
```

Comment:
American English vs British English?

Reply:
First of all: fixing a typo (as AE and BE have been mixed :) )